packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -1486,28 +1486,29 @@ class _TableRowState<T> extends State<TableRow<T>>
         }
         // If ColumnRenderer.build returns null, fall back to the default
         // rendering.
-        content ??= RichText(
-          text: TextSpan(
-            text: column.getDisplayValue(node),
-            children: [
-              if (column.getCaption(node) != null)
-                TextSpan(
-                  text: ' ${column.getCaption(node)}',
-                  style: const TextStyle(
-                    fontStyle: FontStyle.italic,
-                  ),
+        content ??= column.getDisplayWidget(node, context) ??
```

Comment:
once you have your columns implement `ColumnRenderer`, please revert changes to this file